.editorconfig

```diff
@@ -7,11 +7,11 @@ root = true
 end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
-indent_style = space
-indent_size = 2
+indent_style = tab
+indent_size = 4
```

Comment:
Why this change? For JS most of our repos use the 2 space indent, I think it'd be better to stick with that.